war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -692,14 +721,15 @@ function registerValidator(e) {
   var checker = function () {
     const validationArea = this.targetElement;
     FormChecker.sendRequest(this.targetUrl(), {
-      method: method,
-      onComplete: function ({ status, responseText }) {
+      onComplete: function (response) {
```

Comment:
The old code propagated the `method` to the object passed in as the second argument of `FormChecker.sendRequest`. The new code does not.